src/message-manager/AbstractMessageManager.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
Could we leave this rule enabled, but use a type guard to keep the `undefined` check around?

e.g. instead of
```
expect(message).not.toBeUndefined();
if (message) {
  expect(message.status).toBe('rejected');
}
```

We could do this:

```
if (!message) {
  throw new Error('Message undefined')
}
expect(message.status).toBe('rejected');
```

We are allowed to `throw` conditionally, and TypeScript is _supposed_ to be smart enough to infer that after a check like that, the type can't be `undefined` anymore.

Leaving this rule disabled for entire modules seems less than ideal, because it is really easy to write tests that are _completely broken_ if some expectations are conditional.